crates/bdk/tests/wallet.rs

```diff
@@ -139,6 +139,25 @@ fn test_get_funded_wallet_tx_fee_rate() {
     assert_eq!(tx_fee_rate.as_sat_per_vb(), 8.849558);
 }
 
+#[test]
+fn test_list_output() {
+    let (wallet, txid) = get_funded_wallet(get_test_wpkh());
```

Comment:
nit: It would be better if this test also included an unconfirmed transaction, but if you don't want to, that's fine as well

Reply:
I think it should be fine as `filter_chain_txouts` is thoroughly tested?